draft-ietf-quic-v2.md

```diff
@@ -117,19 +117,40 @@ nonce = 0x141b99c239b03e785d6a2e9f
 ~~~
 
 
-# Version Negotiation Considerations
+# Compatible Version Negotiation
 
 QUIC version 2 endpoints SHOULD also support QUIC version 1. Any QUIC endpoint
 that supports multiple versions MUST fully implement {{QUIC-VN}} to prevent
 version downgrade attacks.
 
-Note that version 2 meets that document's definition of a compatible version
-with version 1. Therefore, v2-capable servers MUST use compatible version
-negotiation unless they do not support version 1.
-
-As version 1 support is more likely than version 2 support, a client SHOULD use
-QUIC version 1 for its original version unless it has out-of-band knowledge that
-the server supports version 2.
+QUIC version 2 is symmetrically compatible with QUIC version 1 as defined in
+{{Section 3 of QUIC-VN}}.  That is, a server can choose to negotiate version 2
+after a client initiates a connection using version 1; or, a server can
+negotiate version 1 if a client initiates a version 2 connection attempt.
+
+A server that supports both version 1 and version 2 SHOULD support a compatible
+upgrade to version 2 if a client offers the option.
+
+As the only difference between version 1 and version 2 is the protection of
```

Comment:
Given the differing labels, all the protection is different, no?

Reply:
Good point.